### ISSUE TITLE
Incompleteness due to conditional permission expressions

### ISSUE THREAD

**Issue by viper-admin** [viperproject/silicon#199]
> Created by **@mschwerhoff** on 2015-12-03 15:37

Test `wands\regression\packaging_cond_perm.sil` doesn't verify:

```
#!text

inhale acc(x.f,2/4)
package acc(x.f,2/4) && x.f > 0 --* (acc(x.f, 1/4) && acc(x.f, (x.f > 0 ? 2/4: 1/4)) )

exhale acc(x.f,1/4) // FAILS
```

**Comment by mschwerhoff** [comment]
Self-contained test (aforementioned file no longer exists):

```
field f: Int

method test(x: Ref) {
  inhale acc(x.f,2/4)
  
  package acc(x.f,2/4) && x.f > 0 --* (acc(x.f, 1/4) && acc(x.f, (x.f > 0 ? 2/4: 1/4)) )

  exhale acc(x.f,1/4) // FAILS
}
```

**Comment by mschwerhoff** [comment]
If the conditional is pulled out, then it verifies:

```
  package
    acc(x.f, 2/4) && x.f > 0
      --*
    acc(x.f, 1/4) && (x.f > 0 ? acc(x.f, 2/4) : acc(x.f, 1/4))
```

See also #205

**Comment by mschwerhoff** [comment]
Resolved with commit e9a7c26
